Consistent UI in desktop app download page

Mr-Anonymous75 opened the pull request:
The download buttons are now of the same style as all the other buttons on the page, instead of the retro gradient glossy things. The link for 32-bit windows downloads has been turned into a second button.

The period in the text "It even works when you're offline." has been changed to an exclamation mark.

<sub><sup>Hmm... Github thinks I've deleted 728 lines and added 735, though I really haven't.</sub></sup>

GarboMuffin commented:
Whatever text editor you're using decided to indent everything by two spaces, so please undo that, then I'll look at this more

Mr-Anonymous75 commented:
> Whatever text editor you're using decided to indent everything by two spaces, so please undo that, then I'll look at this more

Umm uhh... Github did that.

Mr-Anonymous75 commented:
> Whatever text editor you're using decided to indent everything by two spaces, so please undo that, then I'll look at this more

Okay... Seems that you did it yourself.

GarboMuffin commented:
Not sure why you closed this and deleted the branch. I fixed the indent problem myself because it seemed like you weren't going to, I haven't made an opinion yet on whether the changes made were good or bad.

Mr-Anonymous75 commented:
> Not sure why you closed this and deleted the branch. I fixed the indent problem myself because it seemed like you weren't going to, I haven't made an opinion yet on whether the changes made were good or bad.

Huh? Did I?

Oh, wait, I renamed it. I'm totally new to pull requests and branches, so I had no idea this would happen. I'll open a new pull request or something.

Mr-Anonymous75 commented:
#202
